### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
In this PR :

- I added HTML and CSS for the contact form.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -302,3 +302,69 @@ button:disabled {
   border-radius: 0;
   color: #000;
 }
+
+.contacts {
+  background-image: url("./images/contactshapes.png");
+  display: flex;
+  background-color: #396df2;
```
Comment (pinkmoon25) [inline style.css]: - [ ] Great work Implementing the project requirements :clap: , however as one of the main goal of the project is to parse a figma template as close to the original design as possible, please use the correct background color for this section it should be ` background-color: #6070FF;`

File: style.css [hunk 2]
```diff
@@ -302,3 +302,69 @@ button:disabled {
...
+  font-weight: 900;
+  color: #fff;
+}
+
+.paragraph {
+  width: 380px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 25px;
```
Comment (pinkmoon25) [inline style.css]: - [ ] Please use the correct properties for fonts such as size line-height and weight, as it is mentioned in the figma template, please remember we have to stick to the design as much as possible (e.g., font, colors, images, text, margins)
from figma :point_down: 
```
font-size: 20px;
font-weight: 400;
line-height: 28px;
text-align: center;

```

- [ ] Please also try to match the figma design for your contact section's paragraph as well as for input fields, see your design :point_down: 

image

see figma design :point_down: 
image

Notice the padding for input fields, and how the text is centered on the page with some right and left padding

File: style.css [hunk 3]
```diff
@@ -302,3 +302,69 @@ button:disabled {
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-me {
+  font-family: "poppins", sans-serif;
+  margin-block-end: 0;
+  font-size: 40px;
+  font-weight: 900;
```
Comment (pinkmoon25) [inline style.css]: - [ ] Please see the font-weight property according to the figma template is 700, Please fix this too 

image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Roland-Ntwali @ngeelani48 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
Pair programming :heavy_check_mark: 
Form implementation :heavy_check_mark: 
Integrate formspree :heavy_check_mark: 
HTML validation :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi Team, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Form submission works

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**